app/app/src/main/java/me/bickositieff/raspio/MainActivity.kt

```diff
@@ -22,5 +24,6 @@ class MainActivity : AppCompatActivity() {
                 R.id.navigation_playlist, R.id.navigation_server_controls, R.id.navigation_notifications))
         setupActionBarWithNavController(navController, appBarConfiguration)
         navView.setupWithNavController(navController)
+        ApiHolder.decorator = ServerURLDecorator("http://10.0.2.2:3000")
     }
 }
```

Comment:
```suggestion
    }
    
    init {
        ApiHolder.decorator = ServerURLDecorator("http://10.0.2.2:3000")
    }
}
```
Init runs even earlier. A race condition could occur when later a ViewModel makes a request when it's loaded.